src/components/SingleReview.jsx

```diff
@@ -1,14 +1,27 @@
 import { useEffect } from "react"
 import { useState } from "react"
 import { useParams } from "react-router"
-import { getSingleReview } from "../api"
+import { getSingleReview, patchVotes } from "../api"
 import { Comments } from "./Comments"
 
 export const SingleReview = ()=> {
     const [isLoading, setIsloading] = useState(true)
     const [review, setReview] = useState([])
+    const [voteChange, setVoteChange] = useState(0)
+
     const {review_id} = useParams()
 
+    const voteChanger = (num) => {
+        setVoteChange((currVoteChange) => {
```

Comment:
Great use of optimistic rendering here so the user doesnt spam the like button while they wait.